### ISSUE TITLE
openshift-logging:  also build eventrouter

### ISSUE THREAD

**Issue by ibotty** [okd-project/okderators-catalog-index#6]
The manual still refers to registry.redhat.io/openshift-logging/eventrouter-rhel9.

**Comment by kai-uwe-rommel** [comment]
It is also available from quay.io/openshift-logging/eventrouter:latest

**Comment by SupremeMortal** [comment]
Closing this as won't fix. AFAIK, this is part of the ELK stack which we don't plan to support as it has been removed in the 6.x line of cluster logging.

**Comment by kai-uwe-rommel** [comment]
No, AFAIK, it is independent of the ELK/EFK stack. Instead, it is a daemon that reads the K8s events to be collected by whatever logging solution is used for a cluster.

**Comment by SupremeMortal** [comment]
Interesting. It's tagged as part of the ELK stack, but maybe that hasn't been updated. I'll include it in my next build of the cluster logging operator.

**Comment by SupremeMortal** [comment]
The event router is built and can be found at this image URL `quay.io/okderators/cluster-logging/eventrouter:latest`

The latest version of the operator does not have a dependency on eventrouter anymore, so you have to follow this guide to install it: https://github.com/openshift/cluster-logging-operator/blob/5bae4ccfc5ef05fdf77a325e54ef14b8e87b6b0a/docs/administration/deploy-event-router.md

**Comment by kai-uwe-rommel** [comment]
@SupremeMortal, you wrote `quay.io/okderators/cluster-logging/eventrouter:latest` but in the linked doc it says "openshift-logging" instead of "cluster-logging" in the path...

**Comment by SupremeMortal** [comment]
Yes... for now that is the reference we have. I'll need to port this documentation to the okd.io website and replace the openshift usages.
